episodes/season_theatrical.yml

```
@@ -3,9 +3,9 @@ episodes:
 - title: The Longest Daycare
   season: 25
   episode: 25
-  release_date:
-  directors:
-  writers:
+  release_date: 2012-07-13
+  directors: David Silverman
```

for integrity reasons these also have to be arrays of strings - the script  is not quite smart enough to take singles and turn them into arrays for these fields. Would you mind bumping these when you get a sec?

```suggestion
  directors: ["David Silverman"]
```